posthog/migrations/0243_unpack_plugin_source_files.py

```diff
@@ -65,9 +65,11 @@ def update_or_create_from_plugin_archive(plugin):
         try:
             update_or_create_from_plugin_archive(plugin)
         except exceptions.ValidationError as e:
-            logger.warn(f"Migration 0243 - failed to extract or save code of plugin {plugin.name} ID {plugin.id}: {e}")
+            logger.warn(
+                f"Migration 0243 - skipping plugin {plugin.name} ID {plugin.id}, failed to extract or save its code: {e}"
+            )
         else:
-            logger.info(f"Migration 0243 - extracted and saved code of plugin {plugin.name} ID {plugin.id}")
+            logger.debug(f"Migration 0243 - extracted and saved code of plugin {plugin.name} ID {plugin.id}")
```

Comment:
Avoid string interpolation when logging.

```suggestion
            logger.debug("Migration 0243 - extracted and saved code of plugin." plugin=plugin.name, plugin_id=plugin.id)
```